M/mlir_jl_tblgen/build_tarballs.jl

```diff
@@ -0,0 +1,117 @@
+using BinaryBuilder, Pkg
+using Base.BinaryPlatforms
+
+const YGGDRASIL_DIR = "../.."
+include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
+include(joinpath(YGGDRASIL_DIR, "platforms", "llvm.jl"))
+
+name = "MLIR_JL_TBLGEN"
+repo = "https://github.com/JuliaLabs/MLIR.jl.git"
+version = v"0.0.1"
+
+llvm_versions = [v"14.0.6", v"15.0.7", v"16.0.6"]
+
+sources = [
+    GitSource(repo, "ff7e7e9f342b5fd1932efbee506d19bec6cf3159")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd MLIR.jl/deps/tblgen
```

Comment:
Shouldn't we move it to another dir? We can do it later, just an opinion

Reply:
I don't think there is a need for it, we could make it follow the structure a bit more, but right now that seems over the top for two cc files.